ext/fetch/lib.rs

```diff
@@ -468,6 +471,190 @@ pub async fn op_fetch_send(
   })
 }
 
+#[op]
+pub async fn op_fetch_send2(
```

Comment:
I think this should be deduplicated with `op_fetch_send`, which means that `fetch()` implementation will need to call `op_fetch_response_into_byte_stream()`.

Reply:
Alternatively a boolean can be passed to `op_fetch_send` to indicate whether we want to have reposnse turned into a byte stream immediately.